README.md

```diff
@@ -138,6 +140,7 @@ The following are the parameters for creating the cloudformation template:
 4. `SenderEmail` : [Required] The SES sending email configured in the [Pre-Requisites](#pre-requisites)
 5. `RecipientEmail` : [Required] Comma separated list of recipient email addresses configured in [Pre-Requisites](#pre-requisites).
 6. `UseVPCAndSubnets` : [Optional] Whether to use the vpc and subnets to create a security group and link the security group and vpc to the lambdas. When UseVPCAndSubnets left out (default) or set to 'true', lambdas are connected to a VPC in your account, and by default the function can't access the RDS (or other services) if VPC doesn't provide access (either by routing outbound traffic to a [NAT gateway](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html) in a public subnet, or having a [VPC endpoint](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-endpoints.html), both of which incur cost or require more setup). If set to 'false', the [lambdas will run in a default Lambda owned VPC that has access to RDS (and other AWS services)](https://docs.aws.amazon.com/lambda/latest/dg/configuration-vpc.html#vpc-internet).
+7. `SlackWebhookUrls` : [Optional] Comma separated list of slack webhooks for failure alerts.
 
```

Comment:
@su7edja additional info be added in the `setup` steps for sam deploy with slack urls and in `How it works` section specifically the alerting support.

Reply:
Thanks for the review @namitad 
I added the steps you mentioned above. Could you add "hacktoberfest-accepted" label to the PR if this is acceptable?